marley_cake_adventure.py

```diff
@@ -0,0 +1,76 @@
+"""
+author: kalez43@github.com
+description: This is a text-based game. The goal is to make the best decisions in order to allow Marley (dog) to
+retrieve a cake off of the counter without Kayleigh (her human) catching her.
+"""
+
+
+# Space between story strings
+def space():
+    print("")
+
+
+def stars():
+    space()
+    print("* * * * *")
+    space()
+
+
+# Enter function between story telling strings
+def enter():
+    space()
+    x = 0
+    while x < 1:
+        x += 1
+        user_input = input("Click here then press enter ==>")
+        if user_input == "":
+            continue
+        else:
+            break
+
+
+# Intro
+space()
+print("Marley's Cake Adventure")
+stars()
+print("""Marley is a very hungry hound! Cake is her favorite food, and Kayleigh has just placed a freshly baked birthday
+cake on the counter to cool. Help Marley retrieve the cake without Kayleigh noticing!""")
+space()
+print("""Click where prompted and press enter to move to the next segment. Answer prompts to give Marley the best 
+chance of succeeding. And don't forget to be quiet!""")
+enter()
+stars()
+print("""You are a medium sized dog named Marley. You are lying on your dog bed watching Kayleigh, your human, bake a 
+cake in the kitchen. Your dog bed is behind the couch and 18 feet away from the kitchen counter.""")
+enter()
+stars()
+print("Kayleigh places the cake on the counter and walks 20 feet away to sit down on the couch.")
+space()
+print("""You notice that she is facing away from the counter, and is reading a book. Her attention should be 
+filled for now.""")
+enter()
+stars()
+
+
+def decision1():
+    print("""You can:

```

Comment:
You are making good progress, using methods and all! Great job! If I were to suggest anything, I'd say remove the 1 from the method name. This from the book Clean Code: https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882

_Methods should have verb or verb phrase names like postPayment, deletePage, or save.
Accessors, mutators, and predicates should be named for their value and prefixed with get,
set, and is according to the javabean standard._ - Clean Code

Reply:
Wait, which part is a method?? decision1()? I still don't fully understand the terminology :/

Reply:
Yes. Method == function